README.md

```diff
@@ -16,14 +16,11 @@ None.
 
 ## Role Variables ##
 
-None.
 
-<!--
 | Variable | Description | Default | Required |
 |----------|-------------|---------|----------|
-| optional_variable | Describe its purpose. | `default_value` | No |
-| required_variable | Describe its purpose. | n/a | Yes |
--->
+| clamav_cron_frequency | Set the cronjob frequency of clamscan. Must be one of : `hourly`,`daily`,`weekly` or `monthly` | `weelky` | No |
+
```

Comment:
Fix typo and tweak language:
```suggestion
| clamav_cron_frequency | The frequency of ClamAV scanning.  Must be one of : `hourly`,`daily`,`weekly` or `monthly`. | `weekly` | No |
```

Reply:
Line 23 above is an extraneous blank line, and I think that is what is making the linters fail.  Can you remove it as well?

Reply:
Outdated.  See [here](https://github.com/cisagov/ansible-role-clamav/pull/37#pullrequestreview-861365215) instead.